locales/it/index.yml

```
@@ -913,7 +913,7 @@ wallet:
     loading: Carico la lista delle banche...
   accessibility:
     sectionCardLabel: "Aggiorna metodi di pagamento"
-    sectionCardHint: "Aggiorna la lista dei metodi di pagamento"
+    sectionCardHint: "tocca due volte per aggiornare la lista"
```

```suggestion
    sectionCardHint: "Aggiorna la lista dei metodi di pagamento"
```
For the same reason of the previous comment.